src/components/InteractionLayer/index.js

```diff
@@ -151,7 +150,7 @@ class InteractionLayer extends React.Component {
     }
 
     if (
-      (!prevProps.width && this.props.width && this.props.ruler.timestamp) || // got width from sizeMe
+      (prevProps.width !== this.props.width && this.props.ruler.timestamp) || // got new width from sizeMe
```

Comment:
Simply `// width changed` is sufficient. Or even omit it since this `if` isn't super complicated to understand.